File: scala/src/main/scala/com/shekhargulati/ninetynine_problems/_01_lists/P24.scala
package com.shekhargulati.ninetynine_problems._01_lists

/**
  * (*) Lotto: Draw N different random numbers from the set 1..M
  */
object P24 {

  def randomSelect(n: Int, range: (Int, Int)): List[Int] = P23.randomSelectR(P22.range(range._1, range._2), n)

}

File: java8/src/main/java/com/shekhargulati/ninetynine_problems/_01_lists/P24.java
package com.shekhargulati.ninetynine_problems._01_lists;

import java.util.List;

/**
 * (*) Lotto: Draw N different random numbers from the set 1..M
 */
public class P24 {

    public static List<Integer> randomSelect(int n, int start, int end) {
        return P23.randomSelect(P22.range(start, end), n);
    }
}
